File: app/src/main/java/com/neome/api/meta/base/dto/DefnLayoutCalendar.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdField;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnLayoutCalendar extends DefnLayoutGrid
{
  @Nullable
  public MetaIdField colorFieldId;

  @Nullable
  public MetaIdField fromDateFieldId;

  @Nullable
  public MetaIdField fromTimeFieldId;

  @Nullable
  public MetaIdField[] showFieldIdSet;

  @Nullable
  public MetaIdField titleFieldId;

  @Nullable
  public MetaIdField toDateFieldId;

  @Nullable
  public MetaIdField toTimeFieldId;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnLayoutCalendar.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdField

interface DefnLayoutCalendar : DefnLayoutGrid {
    val colorFieldId: MetaIdField?
    val fromDateFieldId: MetaIdField?
    val fromTimeFieldId: MetaIdField?
    val showFieldIdSet: List<MetaIdField>?
    val titleFieldId: MetaIdField?
    val toDateFieldId: MetaIdField?
    val toTimeFieldId: MetaIdField?
}
